cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+ steps:
+ - name: maven:3-jdk-11
+   entrypoint: mvn
+   args: ['test']
+   dir: 'capstone/'
+ - name: maven:3-jdk-11
+   entrypoint: mvn
+   args: ['package', 'appengine:deploy']
```

Comment:
Remove the appengine:deploy step. We do not want to deploy every time we push a commit to a PR.

Deployments should be manual until/if we investigate how to deploy on every PR merge.